src/pages/Beta/Airdrop2/QuestionBox/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
+import { Text } from '@pangolindex/components'
+import { StyledLogo, QuestionBox } from '../styleds'
+import PlusLogo from 'src/assets/images/plus.png'
+import MinusLogo from 'src/assets/images/minus.png'
+
+export const QuestionAnswer = () => {
+  const [visible, setVisible] = useState<boolean>(false)
+
+  var axios = require('axios')
+  var data = JSON.stringify({
+    query: `query getKnowledge($filter: kb_filter) {
+      kb(filter: $filter) {
+          id
+          title
+          content
+      }
+  }`,
+    variables: { filter: { category: { _eq: 'Airdrop' } } }
+  })
+
+  var config = {
+    method: 'post',
+    url: 'https://p7gm7mqi.directus.app/graphql',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    data: data
+  }
+
+  const [title, setTitle] = useState<string>('')
+  const [content, setContent] = useState<string>('')
+
+  axios(config).then(function(response: any) {
+    setTitle(JSON.stringify(response.data.data.kb.map((e: any) => e.title)).replace(/[[\]"]/g, ''))
+    setContent(JSON.stringify(response.data.data.kb.map((e: any) => e.content)).replace(/[[\]"]/g, ''))
+  })
+
```

Comment:
this is not a good practice to call axios on every render. please separate this logic into some hook, please refer how we have implemented news apis